From 88fe8dae31162dc346bfefa7a625700cbefae3c4 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Mon, 21 Jan 2013 18:47:46 +0000
Subject: [PATCH 0391/1077] LogService.exe massaiSNMP.dll V1.0.4.2 Mantis
 0025700: LogService: new event-/comp-configuration for british airlines
 BA_CKC

git-svn-id: svn://localhost/SelfServiceCommon/trunk@468 90b65887-3827-0410-9a23-83215b262276
---
 .../APPLICATIONS.BA_CKC.compDefinition.cfg    |  2 ++
 .../APPLICATIONS.BA_CKC.compDefinition.csp    |  2 ++
 .../cpp/LogService/eventConfiguration.txt     | 24 +++++++++++++++++++
 .../cpp/LogService/inc/LogService_def.h       |  9 ++++---
 .../cpp/LogService/inc/massaiSNMP_def.h       |  9 ++++---
 5 files changed, 40 insertions(+), 6 deletions(-)
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.BA_CKC.compDefinition.cfg
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.BA_CKC.compDefinition.csp

diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.BA_CKC.compDefinition.cfg b/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.BA_CKC.compDefinition.cfg
new file mode 100644
index 00000000..2e679eb0
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.BA_CKC.compDefinition.cfg
@@ -0,0 +1,2 @@
+AL_BA_CKC;AVAILABLE=37001 | 37002 | 37003 | 37004 | 37005;UNAVAILABLE=37000 | 37006 | 37007 | 37008 | 37010 | 37011 | 37012
+APP_BA_CKC;AVAILABLE=AL_BA_CKC.AVAILABLE & CLA.AVAILABLE & CUSS_ATB_COMP.OK & CUSS_ATB_FEEDER_BP.OK
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.BA_CKC.compDefinition.csp b/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.BA_CKC.compDefinition.csp
new file mode 100644
index 00000000..f48c0f52
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.BA_CKC.compDefinition.csp
@@ -0,0 +1,2 @@
+ALL_OK;&;APP_BA_CKC.AVAILABLE;APPLICATIONS.BA_CKC.compDefinition
+PARTLY_OK;|;APP_BA_CKC.AVAILABLE;APPLICATIONS.BA_CKC.compDefinition
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt b/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
index 6ce83039..2c2d4677 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
+++ b/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
@@ -1299,6 +1299,30 @@
 36012;1280802,A3_CKC;[STAT][LOG][SNMP:#:1:30];aegean airlines checkin application change state request from unavailable to stopped requested by System Manager
 36013;1050804,A3_CKC;[STAT][LOG][SNMP:#:0:30];aegean airlines checkin application change state from available to active requestet by A3_CKC
 
+37000;1190000,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application initialize
+37001;1040000,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application init ready
+37002;1050801,BA_CKC;[STAT][LOG][SNMP:#:0:30];british airlines checkin application activated
+37003;1060000,BA_CKC;[STAT][LOG][SNMP:#:0:30];british airlines checkin application terminated
+37004;1130000,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application unavailable
+37005;1120000,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application available
+37006;1030001,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application disabled
+37006;1030801,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application disabled
+37006;1010001,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application disabled
+37007;1070000,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application stopped
+37007;1080000,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application stopped
+37007;1090000,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application stopped
+37007;1100000,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application stopped
+37007;1110000,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application stopped
+37008;1070802,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application stopped by SM
+37008;1080802,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application stopped by SM
+37008;1090802,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application stopped by SM
+37008;1100802,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application stopped by SM
+37008;1110802,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application stopped by SM
+37009;1060801,BA_CKC;[STAT][LOG][SNMP:#:0:30];british airlines checkin application terminated
+37010;1300000,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application changed state from available to unavailable
+37011;1290000,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application changed state from initialize to unavailable
+37012;1280802,BA_CKC;[STAT][LOG][SNMP:#:1:30];british airlines checkin application change state request from unavailable to stopped requested by System Manager
+37013;1050804,BA_CKC;[STAT][LOG][SNMP:#:0:30];british airlines checkin application change state from available to active requestet by BA_CKC
 
 
 9000;10000,CONVEYOR;[STAT][LOG][SNMP:#:1:30];Conveyor available
diff --git a/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h b/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h
index 09cd36a1..3b287a64 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h
+++ b/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h
@@ -11,8 +11,8 @@
 
 #include "Tools/VERSION.H"
 
-#define FILE_VER_BIN     1,0,4,1
-#define FILE_VER_STR     "FileVersion", "1.0.4.1\0"
+#define FILE_VER_BIN     1,0,4,2
+#define FILE_VER_STR     "FileVersion", "1.0.4.2\0"
 
 /*!
 \page LogService.exe - Logging Service
@@ -22,10 +22,13 @@
 
 !!! immer auch wenn nötig massaiSNMP_def.h pflegen !!!
 
-@version 1.0.4.1
+@version 1.0.4.2
 
 \section change1 Change history
 
+\date 21.01.2013 \li V1.0.4.2
+                 \li SteT Mantis 0025700: LogService: new event-/comp-configuration for british airlines BA_CKC
+
 \date 18.01.2013 \li V1.0.4.1
                  \li SteT Mantis 0025600: MR SankyoICM300: DeviceHandler for MCR
                  \li added GmrDev_AccessIS6x0
diff --git a/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h b/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h
index e2cd6020..7640f2d4 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h
+++ b/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h
@@ -11,8 +11,8 @@
 
 #include "Tools/VERSION.H"
 
-#define FILE_VER_BIN     1,0,4,1
-#define FILE_VER_STR     "FileVersion", "1.0.4.1\0"
+#define FILE_VER_BIN     1,0,4,2
+#define FILE_VER_STR     "FileVersion", "1.0.4.2\0"
 
 /*!
 \page massaiSNMP.dll - plugin for MS SNMP-Service;
@@ -22,10 +22,13 @@
 
 !!! immer auch wenn nötig logService_def.h pflegen !!!
 
-@version 1.0.4.1
+@version 1.0.4.2
 
 \section change1 Change history
 
+\date 21.01.2013 \li V1.0.4.2
+                 \li SteT Mantis 0025700: LogService: new event-/comp-configuration for british airlines BA_CKC
+
 \date 18.01.2013 \li V1.0.4.1
                  \li SteT Mantis 0025600: MR SankyoICM300: DeviceHandler for MCR
                  \li added GmrDev_AccessIS6x0
-- 
2.41.0.windows.1

